extensions/entry-extension-collapsible/README.md

```diff
@@ -4,27 +4,37 @@ This extension demonstrates how you would use all existing field editors to crea
 
 ![demo](./docs/demo.gif)
 
-## Commands
+## Available Scripts
```

Comment:
curious if the changes here are relevant? They seem very CCA based. Plus we say `npm` below but there's also a yarn file?

Reply:
I kept the original README instruction from CCA + added the description text for the extension from the old README. So the commands are relevant I think, except you are right that we should `yarn` there instead (and also description for `upload` script is missing).

Do you think adding description for `upload` and changing the commands to use `yarn` instead of `npm` would solve the issue? :)

Reply:
Updated some things here :)